cypress/support/step_definitions/annual-review/scheduler/annual_review_date_was_int_days_ago.js

```diff
@@ -0,0 +1,70 @@
+import { subDays, startOfDay, addDays, subYears } from "date-fns";
+
+When("annual review date was {int} days ago", async (daysAfterAnnualReview) => {
+  const annualReviewDate = startOfDay(subDays(new Date(), daysAfterAnnualReview))
+
+  const INTERVALS_IN_DAYS = {
```

Comment:
How about 
1. setting the annual review date to 41 days in the past (pretty sure this will work..)
2. run the batch process so you don't have to calculate these yourself (it also means it will always match the batch process logic) `when the batch process runs` 
3. then run the worker process `when the worker process runs`

the other thing you can do is 
1. set it for tomorrow
2. run batch process 
3. use a step to change the jsonData. All of it will already be generated, you just have to set keyDates.unpublished.UNPUBLISH to todays date 

you want to change .UNPUBLISH since `/workers/unpublish/dayBefore/getMetaForList.ts` is actually just looking for unpublish date and subtracting one day.